executor/common_linux.h

```diff
@@ -5274,3 +5274,23 @@ static long syz_clone3(volatile long a0, volatile long a1)
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_sched_setattr
+
+#include <linux/sched.h>
+
+struct sched_attr_prefix {
+	__u32 size;
+	__u32 sched_policy;
+};
+
+static long syz_sched_setattr(volatile long pid, volatile long attr_ptr, volatile long flags)
+{
+	struct sched_attr_prefix* attr = (struct sched_attr_prefix*)attr_ptr;
+	// Don't assign real time sched policies as it leads to numerious false positive stalls.
+	if (attr->sched_policy == SCHED_FIFO || attr->sched_policy == SCHED_RR)
```

Comment:
If it's just to prohibit some values, do it here:
https://github.com/google/syzkaller/blob/master/sys/linux/init.go#L167-L239
we can't have a pseudo-syscall for each real syscall just to filter out values.

Reply:
Here the problem was that the policy is not passed to the syscall as a separate argument, but rather as field in a struct, which lies somewhere in memory. Nothing prevents syzkaller from squashing it into a blob and mutating it as it wishes.

I could fit it in `neutralize` if I just discard (replace by 0) the second argument if it is not a pointer to a struct. What do you think?

Reply:
Yes. Do whatever is simpler code-wise.
(don't forget to add a test)